### ISSUE TITLE
information

### ISSUE THREAD

**Issue by Elisa0122** [Elisa0122/svg-icon-sets#1]
hi @wendywarren,
Could you explain me how to overwrite an svg by using css please?
To pass Markbot I just changed the fill in my svg file but I didn't find the way by using css.
thank you
Elisa

**Comment by wendywarren** [comment]
Hi @Elisa0122,

I apologize. Your issue got lost in a pool of first year issues with creating their first website and first issue.

Did today's class answer your question?

Wendy

**Comment by Elisa0122** [comment]
Yes thanks I didn't minus on illustrator I think it was for that. I know it pass through Markbot so I didn't want to try to make changes at what I did already.
Now I just have to make my project done and I did all the assignments.
Do you know what we will learn for web design 3? With my 2 kids during summer break I would like to have to work on it earlier if I can.

Thank you

Elisa

On Feb. 18, 2022 14:06, Wendy Warren ***@***.***> wrote:

Hi @Elisa0122<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FElisa0122&data=04%7C01%7Cnico0122%40algonquinlive.com%7C1bc1e656ece041c0a5df08d9f311ba85%7Cec1bd9240a6a4aa9aa89c980316c0449%7C0%7C0%7C637808079738719282%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=4K%2FLZAkl9jFpf5amRMa3xtZT2oNsC96JHawS2Iv53JQ%3D&reserved=0>,

I apologize. Your issue got lost in a pool of first year issues with creating their first website and first issue.

Did today's class answer your question?

Wendy

—
Reply to this email directly, view it on GitHub<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FElisa0122%2Fsvg-icon-sets%2Fissues%2F1%23issuecomment-1045031113&data=04%7C01%7Cnico0122%40algonquinlive.com%7C1bc1e656ece041c0a5df08d9f311ba85%7Cec1bd9240a6a4aa9aa89c980316c0449%7C0%7C0%7C637808079738719282%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=JbJGxddoTxx7dEeP31pFYD9iuriPwZ3TAfjWCqd5sNI%3D&reserved=0>, or unsubscribe<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAT7G3PJ4RCYMUWPOIH254O3U32KCHANCNFSM5NSGHVMA&data=04%7C01%7Cnico0122%40algonquinlive.com%7C1bc1e656ece041c0a5df08d9f311ba85%7Cec1bd9240a6a4aa9aa89c980316c0449%7C0%7C0%7C637808079738719282%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=L6R0mLASVLLqfse8PlaxF%2BRhSsqL4gRH3YjtTLZzfrc%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cnico0122%40algonquinlive.com%7C1bc1e656ece041c0a5df08d9f311ba85%7Cec1bd9240a6a4aa9aa89c980316c0449%7C0%7C0%7C637808079738719282%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=%2FR0fgDUyKB%2FPplxBYdyx30omsZwb4ExdSoyxJMRPvV4%3D&reserved=0> or Android<https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cnico0122%40algonquinlive.com%7C1bc1e656ece041c0a5df08d9f311ba85%7Cec1bd9240a6a4aa9aa89c980316c0449%7C0%7C0%7C637808079738719282%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=qzoMqS0qupr88lMl1fZaPqsuGSTVyrioB0S3VNwRJpE%3D&reserved=0>.
